### ISSUE TITLE
How do you walk this graph?

### ISSUE THREAD

**Issue by astanciu** [jriecken/dependency-graph#40]
There should be methods to get a node's immediate dependencies or dependents; both of these currently return the full chain (optionally without the last leaf). This makes it hard to walk the graph, step by step.

**Comment by jriecken** [comment]
There isn't currently a method that returns the direct dependencies/dependents, but as a workaround you can currently access the `outgoingEdges ` (dependencies) or `incomingEdges` (dependents) properties of the graph instance. Both are objects where the key is the node name and the value is an array of node names.

```javascript
const g = new DepGraph()
g.addNode('one')
g.addNode('two')
g.addNode('three')
g.addDependency('one', 'two')
g.addDependency('one', 'three')
console.log(g.outgoingEdges['one']) // ['two', 'three']
```

**Comment by jriecken** [comment]
I added `directDependenciesOf` and `directDependantsOf` methods and released a new version (`0.10.0`)
